config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
The only workaround i can think right now to adhere to DRY as well as tackle is above use case is having the below line in base_config:

`lua-load /marathon-lb/getconfig.lua{LOAD_VHOST_MODULE}`
and then in marathon_lb

```
if haproxy_map:
    pass LOAD_VHOST_MODULE = "\n  lua-load /marathon-lb/getvhostmap.lua"
else:
   pass LOAD_VHOST_MODULE = ""
```


And similarly for endpoint Acl. Does that work?